### PR TITLE
Cmuformat

### PR DESCRIPTION
**Pull request by MihaiSurdeanu**

### CODE REVIEW

**Comment by myedibleenso** [comment]
I wonder if I should add `Source` and `Destination` as some kind of modification when converting a Translocation mention into a `SimpleEvent` `EER`.

File: assembly/src/main/scala/org/clulab/reach/assembly/export/AssemblyExporter.scala [hunk 1]
```diff
@@ -423,6 +468,30 @@ object AssemblyExporter {
   val CONTEXT_CELLULAR_COMPONENT = "CONTEXT (CELLULAR COMPONENT)"
   val CONTEXT_TISSUE_TYPE = "CONTEXT (TISSUE TYPE)"
   val TRIGGERS = "TRIGGERS"
+  val TRANSLOCATION_SOURCE = "TRANSLOCATION (SOURCE)"
+  val TRANSLOCATION_DESTINATION = "TRANSLOCATION (DESTINATION)"
+
+  // columns for the CMU tabular format
```
Comment (myedibleenso) [inline assembly/src/main/scala/org/clulab/reach/assembly/export/AssemblyExporter.scala]: Maybe these should be under export/src/main/scala/org/clulab/reach/export/cmu/CMUExporter.scala?

Comment (MihaiSurdeanu) [inline assembly/src/main/scala/org/clulab/reach/assembly/export/AssemblyExporter.scala]: These are used in the "arizona" format.

File: assembly/src/main/scala/org/clulab/reach/assembly/export/AssemblyExporter.scala [hunk 2]
```diff
@@ -451,6 +523,29 @@ object AssemblyExporter {
     AssemblyExporter.SEEN_IN
   )
 
+  val CMU_COLUMNS = Seq(
```
Comment (myedibleenso) [inline assembly/src/main/scala/org/clulab/reach/assembly/export/AssemblyExporter.scala]: ? `CMU_COLUMNS` doesn't appear to be used in `AssemblyExporter`...

Comment (MihaiSurdeanu) [inline assembly/src/main/scala/org/clulab/reach/assembly/export/AssemblyExporter.scala]: They will be soon. Working on it.

**Pull request closed without merging**